src/Appium.Net/Appium/AppiumDriver.cs

```diff
@@ -227,6 +227,11 @@ public void Rotate(Dictionary<string, int> opts)
         public void RemoveApp(string appId) =>
             Execute(AppiumDriverCommand.RemoveApp, AppiumCommandExecutionHelper.PrepareArgument("appId", appId));
 
+        public void TerminateApp(string appId, int? timeout = null) =>
+            Execute(AppiumDriverCommand.TerminateApp, 
+                AppiumCommandExecutionHelper.PrepareArguments(new string[] { "appId", "options" }, 
+                    new object[] { appId, new Dictionary<string, int?>() { { "timeout", timeout } } }));
```

Comment:
the timeout should not be included if it is not set

Reply:
Generally the dictionary item types should be `<String, Any?>`

Reply:
Would you suggest an overload like for BackgroundApp instead of `int? timeout = null`?

Reply:
Unfortunately I cannot advice nothing on this, since I don't mental C# client. Prefer the way, which is already common for other methods, so they all follow the single style.